imdclient/tests/test_imdclient.py

```diff
@@ -148,3 +150,49 @@ def test_pause_resume_no_disconnect(self, server_client_two_frame_buf):
             client.get_imdframe()
         # server should receive disconnect from client (though it doesn't have to do anything)
         server.expect_packet(IMDHeaderType.IMD_DISCONNECT)
+
+    # Default wait_after_disconnect behavior with value set to None
+    def test_unchanged_wait_after_disconnect(self, server_client):
+        """Client ends connection before server sends all frames and continues 
+        default wait behavior on server side
+        """
+        server, client = server_client
+        server.send_frames(0, 2)
+        client.get_imdframe()
+        client.get_imdframe()
+        # client stops connection before server sends all frames
+        client.stop()
+        # server receives only disconnect from client
+        server.expect_packet(IMDHeaderType.IMD_DISCONNECT)
+
+    # wait_after_disconnect behavior with value set to True
+    @pytest.mark.parametrize("wait_after_disconnect", [True])
```

Comment:
It seems like you are using parameterization with only one parameter. I think this test and the next one can be combined into one test that is parameterized with True and False